crates/weaver_forge/src/extensions/code.rs

```diff
@@ -449,6 +465,68 @@ it's RECOMMENDED to:
 		 */"##
         );
 
+        // Test with the optional parameter `line_length=30`
+        // TODO - Figure out where extra space is coming from li/code near bottom.
+        let observed_comment = env
+            .render_str("{{ note | comment(line_length=30) }}", &ctx)
+            .unwrap();
+        assert_string_eq!(
+            &observed_comment,
+            r##"/**
+ * The {@code error.type}
+ * SHOULD be predictable, and
+ * SHOULD have low cardinality.
+ *
+ * <p>
+ * When {@code error.type} is
+ * set to a type (e.g., an
+ * exception type), its
+ * canonical class name
+ * identifying the type within
+ * the artifact SHOULD be used.
+ *
+ * <p>
+ * Instrumentations SHOULD
+ * document the list of errors
+ * they report.
+ * <p>
+ * The cardinality of
+ * {@code error.type} within
+ * one instrumentation library
+ * SHOULD be low. Telemetry
+ * consumers that aggregate
+ * data from multiple
+ * instrumentation libraries
+ * and applications should be
+ * prepared for
+ * {@code error.type} to have
+ * high cardinality at query
+ * time when no additional
+ * filters are applied.
+ * <p>
+ * If the operation has
+ * completed successfully,
+ * instrumentations SHOULD NOT
+ * set {@code error.type}.
+ * <p>
+ * If a specific domain defines
+ * its own set of error
+ * identifiers (such as HTTP or
+ * gRPC status codes), it's
+ * RECOMMENDED to:
+ * <p>
+ * <ul>
+ *   <li>Use a domain-specific
+ *   attribute
+ *   <li>Set {@code error.type}
+ *    to capture all errors,
```

Comment:
This is indeed strange.

Reply:
I need to more formally pull words and remember prefix/postfix space.  The library allows this and uses it, it just doesn't understand *changing* the line prefix over time.


I hope we can, eventually, fix this all up, I just ran out of time to tinker